### ISSUE TITLE
Liquid error: Liquid variables are disabled

### ISSUE THREAD

**Issue by setevoy2** [thepracticaldev/dev.to#2438]
**Describe the bug**
When adding a new post with an Ansible's templates examples - have the _"Liquid error: Liquid variables are disabled"_ error when saving.

**To Reproduce**

1. Go to 'Write a post'
2. Write any post containing Jinja's templating variables, e.g. `{{ my_var }}` (used in Ansible)
3. Press Save post button
4. See the _"Liquid variables are disabled"_ error

**Expected behavior**
A post has to be saved.

**Additional context**
The [draft's URL](https://dev.to/articles/100380).

**Comment by rhymes** [comment]
Nice catch! The error should probably be more "informative" or it should auto-escape those variables statements. Did you escape the templating variables or just inlined them?

**Comment by setevoy2** [comment]
@rhymes Added screenshot.

> Did you escape the templating variables or just inlined them?

Nope - just copy-pasted from the template.
Can try to add an escape "\" symbols - but there is really a lot of such variables...

**Comment by rhymes** [comment]
@setevoy2 no you actually escaped them, they are in a fenced code block. They should be ignored by the parser. Great catch!

**Comment by setevoy2** [comment]
OK, sorry - my bad.
One template wasn't enclosed with the "```".

**Comment by Zhao-Andy** [comment]
Hey @setevoy2, not sure if this is still an issue for you?

Agreed that we could add better error messaging there though.

**Comment by setevoy2** [comment]
@Zhao-Andy No, all works great - it was my mistake with the template's code escaping, thanks.
This issue can be closed.

**Comment by GeraldCrescione** [comment]
Hi team
is there a status update on this bug? It just happened to me too. I can't seem to make the [workaround described here with the backticks](https://dev.to/kritner/liquid-error-help-2h1a) to work

**Comment by SirPhemmiey** [comment]
It's still happening even after 4 years! 

Do we have any update on this?

**Comment by JessicaHu03** [comment]
Same here! We have everything in standard markdown form and have \ before all the variables that need to escape but we still get this error. So confusing.
